manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.MSKLC
+PackageVersion: "1.4"
+InstallerType: exe
+Installers:
+- InstallerUrl: https://download.microsoft.com/download/6/f/5/6f5ce43a-e892-4fd1-b9a6-1a0cbb64e6e2/MSKLC.exe

```

Comment:
This appears to be a self-extracting archive which requires user interaction to install. Do you know if there are any silent switches?